mailpoet/lib/EmailEditor/Integrations/MailPoet/ProductCollection/ProductCollectionEmailRendererRegistrar.php

```
@@ -0,0 +1,96 @@
+      $blockType = $registry->get_registered($blockName);
+      if (!$blockType) {
+        continue;
+      }
+
+      $currentCallback = get_object_vars($blockType)[$renderEmailCallbackProperty] ?? null;
+      if ($this->usesWooCommerceEmailEditorRenderer($currentCallback)) {
+        continue;
+      }
+
+      // @phpstan-ignore-next-line -- WooCommerce email editor reads this dynamic block setting.
+      $blockType->{$renderEmailCallbackProperty} = [$renderer, 'render_block'];
```

PHPStan (2.2.2)

[error] No error to ignore is reported on line 74.


(ignore.unmatchedLine)